meson_options.txt

```diff
@@ -364,3 +364,5 @@ option('llvm-fuzz', type : 'boolean', value : 'false',
        description : 'build against LLVM libFuzzer')
 option('kernel-install', type: 'boolean', value: 'true',
        description : 'include kernel-install feature')
+option('analyze', type: 'boolean', value: 'true',
+       description : 'include systemd-analyze')
```

Comment:
... and this "install systemd-analyze".

"include" is very vague.

Reply:
I agree, I should have specified this a bit better.
Pushed to https://github.com/systemd/systemd/commit/32339800ced9b94241109d5670cfc067f2c92660, should be good now.